.tool-versions

```diff
@@ -1 +1,2 @@
-nodejs 16.14.2
+nodejs ^18.0.0
```

Comment:
I think its fine to pin the same version as the one you defined in the Prerequisites of your README file 😄

Reply:
updated nodejs version to 18.13.0 LTS 😄